packages/@cdktf/hcl2cdk/test/convertProject.test.ts

```diff
@@ -71,13 +81,13 @@ app.synth();`,
           "node": ">=14.0"
         },
         "dependencies": {
-          "cdktf": "0.11.0-pre.53",
+          "cdktf": "latest",
```

Comment:
Shouldn't this also point to the dist directory (or similar) or does it not matter as the code is not synthesized?

Reply:
Yes, but I couldn't get it working the first try and it was not ciritcal to this change so I think we can do a follow-up